config/jobs/kubernetes-sigs/kubebuilder/kubebuilder-presubmits.yaml

```diff
@@ -31,7 +31,7 @@ presubmits:
         command:
         # see https://github.com/kubernetes/test-infra/blob/bb41c9ee91b50fffa843e2a6fbc0fe361999f682/config/prow/config.yaml#L590-L613
         - runner.sh
-        - test_e2e.sh
+        - test_e2e_v1.sh
```

Comment:
I mis-remembered it.
We currently only have one script at https://github.com/kubernetes-sigs/kubebuilder/blob/master/test_e2e.sh.